03_letters/n0npax/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+)
+
+func letters(s string) (freqMap map[rune]int) {
+	freqMap = map[rune]int{}
+	ss := []rune(s)
+	for i := range ss {
+		freqMap[ss[i]]++
+	}
+	return
```

Comment:
Interesting, I haven't seen this too often. I would normally go down the path of explicitly returning the `freqMap` variable as I feel it helps with readability, and is clear what you are returning. But I don't hate, it is growing on me.

Reply:
Avoid named result parameters and naked returns unless you have got good reason  (source [golang wiki](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)) - which I believe you don't here.

Reply:
fixed. Will avoid this in the future. Thanks!